sensor_msgs/msg/Range.msg

```diff
@@ -37,3 +37,7 @@ float32 range           # range data [m]
                         # (Detection too close to the sensor to quantify)
                         # +Inf represents no detection within the fixed distance.
                         # (Object out of range)
+
+float32 variance        # variance of the range sensor
+                        # 0 is interpreted as variance unknown
+
```

Comment:
Small nit: no newline at the end of the file.

Reply:
Hi! Thanks for reviewing my PR. I just removed the newline at the end of the file in the latest commit.
(sorry for the force-push I keep forgetting to sign-off with `git commit -s`)